### PR TITLE
Add Flag feature for important clients

### LINKED ISSUE #8: As an Expert User, I can flag clients for urgent meetings
so that I will not forget important meetings

### PR DESCRIPTION
**Pull request by j4ck990**
Persons object lack flag feature for important clients. 

Important clients require an attribute to distinguish them. 
This closes #8. 

Let's, 
- add Flag class
- add Flag variable for Person class
- add Flag command to edit flag status

Users can now flag clients deemed as important. 

Note: 
- Additional test cases for flag not implemented

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Flag.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    /**
+     * Constructs a {@code Flag} with a boolean.
+     *
+     * @param bool If the person is flagged.
+     */
+    public Flag(String bool) {
+        if (bool.equals("false")) {
+            isFlagged = false;
+        } else if (bool.toLowerCase().equals("true")) {
```
Comment (AD-NAP) [inline src/main/java/seedu/address/model/person/Flag.java:25]: May I know what's the reason why it requires toLowerCase here and not for false?

Comment (j4ck990) [reply src/main/java/seedu/address/model/person/Flag.java:25]: Oh that is to accept case-insensitive input for "f/" prefix. 
So TRUE, True, true are all accepted.

Comment (AD-NAP) [reply src/main/java/seedu/address/model/person/Flag.java]: Does that mean false check needs to be lowercased too?

Comment (AD-NAP) [inline src/main/java/seedu/address/model/person/Flag.java]:
```suggestion
public Flag(String bool) {
        if (bool.toLowerCase().equals("false")) {
            isFlagged = false;
        } else if (bool.toLowerCase().equals("true")) {
```

Comment (j4ck990) [reply src/main/java/seedu/address/model/person/Flag.java]: yup i edited the isValidFlag to check after lowercased as well after the latest commit

**Comment by AD-NAP** [comment]
toLowerCase is now consistent. 
LGTM! 😄

**Pull request merged**